lib/resources.js

```diff
@@ -118,7 +118,9 @@ var parseSearchParams = function (params) {
     sort: { type: 'string', range: Object.keys(SORT_FIELDS) },
     sort_direction: { type: 'string', range: ['asc', 'desc'] },
     search_scope: { type: 'string', range: Object.keys(SEARCH_SCOPES), default: 'all' },
-    filters: { type: 'hash', fields: FILTER_CONFIG }
+    filters: { type: 'hash', fields: FILTER_CONFIG },
+    items_size: { type: 'int' },
+    items_from: { type: 'int' }
```

Comment:
Sorry just noticed we're not sanitizing these. We shouldn't allow `items_size` to be -1 or 100000 for example. We should also set a sensible default so that a consumer leaving off `items_size` doesn't lead to a timeout. I'm thinking...

```
    items_size: { type: 'int', default: 100, range: [0, 100] },
    items_from: { type: 'int', default: 0 }
```

Reply:
makes sense. I've set the upper limit to `200` instead of `100` since right now we've only reindexed bibs with more than `200` items